pkg/flexadapter/nodeserver.go

```diff
@@ -49,6 +49,34 @@ func mountDevice(devicePath, targetPath, fsType string, readOnly bool, mountOpti
 	return diskMounter.FormatAndMount(deviceID, targetPath, fsType, options)
 }
 
+func (ns *nodeServer) waitForAttach(req *csi.NodePublishVolumeRequest, fsType string) error {
+
+	var deviceID string
+
+	if req.GetPublishVolumeInfo() != nil {
+		var ok bool
+		deviceID, ok = req.GetPublishVolumeInfo()[deviceID]
```

Comment:
this looks odd, is `deviceID` an empty string (as in var) or a previously defined?